src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,46 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { Post } from '../../types/Post';
+
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
```

Comment:
make a destructuring
```suggestion
export const PostInfo: React.FC<Props> = ({ post }) => {
  const { ... } = post;
```